src/middleware/globalErrorHandler/middleware.ts

```diff
@@ -0,0 +1,8 @@
+import { Request, Response, NextFunction } from "express";
+
+function main(error: Error, _request: Request, response: Response, _next: NextFunction) {
+  console.log(JSON.stringify(error, Object.getOwnPropertyNames(error)));
```

Comment:
Could we do this like so instead?
```suggestion
  console.error(JSON.stringify(error, Object.getOwnPropertyNames(error)));
```

Reply:
Absolutely!